scripts/download/tools.mjs

```diff
@@ -229,6 +229,45 @@ export default async function main(platform) {
       expectedChecksum:  steveSHA,
       checksumAlgorithm: 'sha512'
     });
+
+  // Download Rancher Dashboard
+  const rancherDashboardVersion = 'v2.6.3-desktop.12';
+  const rancherDashboardURLBase = `https://github.com/rak-phillip/dashboard/releases/download/${ rancherDashboardVersion }`;
```

Comment:
I assume this will eventually move to the `rancher/dashboard` repo, once all the things are done. Otherwise, we should fork it into `rancher-sandbox/dashboard-for-desktop` or something to avoid having things in personal repos.

Reply:
Yes, please move the repo over into the `rancher-sandbox` organization! Let me know if there is a problem moving it, and we can ask Jeff or Matt for help if we don't have the permissions between ourselves.

Reply:
Yes, the intent is to get our builds from `rancher/dashboard` so we don't need to maintain a fork; that's the sole reason this PR is still in draft. If I don't have an answer related to my github actions permissions issues on that end, I'll just fork into `rancher-sandbox` until we get a resolution.

Reply:
Opting to go the `rancher-sandbox` route as I work out issues for `rancher/dashboard` repo.